samples/sensor/pressure_polling/sample.yaml

```
@@ -0,0 +1,9 @@
+sample:
+  name: Pressure polling sample
+tests:
+  sample.sensor.pressure_sensor:
+    build_only: true
+    tags: sensors
+    filter: dt_alias_exists("pressure_sensor")
```

This currently doesn't run. Needs the following change, but also keep only of the two overlay and name it `nrf52dk_nrf52832.overlay` (otherwise neither gets picked up, hence no pressure-sensor alias is found, hence test is filtered :) )

```suggestion
    filter: dt_alias_exists("pressure-sensor")
```

---

> This currently doesn't run. Needs the following change, **but also keep only of the two overlays and name it nrf52dk_nrf52832.overlay (otherwise neither gets picked up, hence no pressure-sensor alias is found, hence test is filtered :) )**

not addressed